data_spec_validator/spec/defines.py

```diff
@@ -195,6 +199,10 @@ def __init__(self, checks, optional=False, allow_none=False, op=CheckerOP.ALL, e
         self._ensure(kwargs)
         if extra:
             print('[DSV][WARNING] keyword: extra is gonna be deprecated')
+        deprecating_checks = {KEY_COEXISTS, ANY_KEY_EXISTS}
+        for deprecating_check in deprecating_checks.intersection(set(checks)):
+            print(f'[DSV][WARNING] Check: {deprecating_check.upper()} is gonna be deprecated')
```

Comment:
I ... don't have any thoughts for now.

What I can think of is to make the checks a CustomEnum. e.g.
```
class SomeEnum:
    INT = 'int', 'stable'
    BOOL = 'bool', 'stable'
    JSON = 'json', 'deprecating'
```
Then the supporting status of each check is much clear.

`extra` is a keyword for the `__init__` constructor, I suppose it can live longer for capability.
Any thoughts ?

Reply:
follow-up,  https://hardcoretech.atlassian.net/browse/PLM-422